service/src/test/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsServiceTest.java

```diff
@@ -143,6 +127,46 @@ void shouldBuildExpectedC2DocumentBundle() {
         assertThat(actual.getC2DocumentBundle().getApplicantName()).isEqualTo(APPLICANT_NAME);
 
         assertC2DocumentBundle(actual.getC2DocumentBundle(), supplement, supportingEvidenceBundle);
+
+        verifyNoInteractions(conversionService);
+    }
+
+    @Test
+    void shouldOnlyConvertApplicationsWhenNotHMCTS() {
+        given(user.isHmctsUser()).willReturn(false);
```

Comment:
would this name make it clearer : shouldOnlySealApplicationWhenNotHMCTS?

Reply:
I guess that they are sort of the inverse of each, this test name implies that we are testing that only the conversion service is called where as the name you are suggesting is implying that anything but the sealing service is called